### PR TITLE
Complete the capstone project

### PR DESCRIPTION
**Pull request by jubaer919**
- **Pages complete**
  - project has home page
  - project has about 

- the project is responsive
- it interact withe two pages
- here is the short video link for the project:
https://www.loom.com/share/f25f16db31ea45a19c257b7243f1b83c

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ After you're finished please remove all the comments and instructions!
```
Comment (arslanbisharat) [inline README.md:97]: - [ ] Please make sure the video contains a highlight about a piece of the code the you finds interesting or is proud of making.

File: README.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (arslanbisharat) [inline README.md:98]: - [ ] 1. The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            <li class="social-items s-logo"><a href="#" class="social-link"><img src="./images/facebook.svg" alt="facebook-logo"></a></li>
+            <li class="social-items s-logo"><a href="#" class="social-link"><img src="./images/s-twitter.svg" alt="twiteer-logo"></a></li>
+            <li class="social-items"><a href="#" class="social-link">English</a></li>
+            <li class="social-items"><a href="#" class="social-link">Mypage</a></li>
+            <li class="social-items"><a href="#" class="social-link">Logout</a></li>
+        </ul>
+    </section>
+    <header class="header">
+        <a href="#" class="logo"><img src="./images/logo.svg" alt="brand logo"></a>
+        <div class="hamburger">
```
Comment (arslanbisharat) [inline about.html:24]: - [ ] Please make sure the background is transparent or the same color/pattern as the top section background.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    <section class="social">
+        <ul class="social-list">
+            <li class="social-items s-logo"><a href="#" class="social-link"><img src="./images/facebook.svg" alt="facebook-logo"></a></li>
+            <li class="social-items s-logo"><a href="#" class="social-link"><img src="./images/s-twitter.svg" alt="twiteer-logo"></a></li>
+            <li class="social-items"><a href="#" class="social-link">English</a></li>
+            <li class="social-items"><a href="#" class="social-link">Mypage</a></li>
+            <li class="social-items"><a href="#" class="social-link">Logout</a></li>
+        </ul>
+    </section>
+    <header class="header">
```
Comment (arslanbisharat) [inline about.html:22]: - [ ] When clicking the menu button on mobile screen, it doesn't work. Please make changes so it works fine.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @jubaer919, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @jubaer919  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [OPTIONAL] You did an excellent job with the mobile menu :+1:, but I believe it would be better if users cannot access the elements from the main page while the menu is open. This can be achieved by using the `position: fixed` CSS property, which fixes the position of the menu on the screen, making it unmovable while it's open. This approach can improve the usability of the mobile menu by preventing accidental clicks on the elements from the main page while the menu is open, which can cause frustration for the user. Additionally, using `position: fixed` can make the menu more accessible and user-friendly, as users will be able to navigate the menu without any distractions or obstacles on the screen. Overall, it's recommended to implement this change to enhance the user experience and usability of the mobile menu.


https://user-images.githubusercontent.com/105117832/235212549-ff97386b-26ca-4dcf-b2f2-4564226fa530.mp4



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**